database/schema.sql

```diff
@@ -0,0 +1,53 @@
+DROP DATABASE IF EXISTS  lululemon;
+CREATE DATABASE lululemon;
+USE lululemon;
+
+-- DROP TABLE IF EXISTS reviews;
+CREATE TABLE products (
```

Comment:
In your review, will you allow the user to choose whether or not the review is helpful for them? If so, will you store that review answer?